### ISSUE TITLE
Keyboard navigation on PhET webpage

### ISSUE THREAD

**Issue by aaronsamuel137** [phetsims/website#41]
During 12/4 developer meet it was brought up that accessibility for the PhET webpage should be addressed. For now, @ariel-phet has asked that the new front-page be reviewed with accessibility concerns in mind. I'll do a pass over the code with this in mind and then assign to @jonathanolson for review.

A question for @ariel-phet first:

Do we care about the visible highlighting of links (i.e. the blue border that shows up in Chrome)? It looks like many links are actually selected but no blue outline shows up (for instance the CU-logo and the HTML5 sims link). I'm guessing this is because these elements have position: absolute. Clearly, blind people won't be able to see the blue outline anyway, but is there any other accessibility concern around have the outline show up?

If you're interesting in seeing what element is selected, open up the developer console and copy and paste this code:

```js
document.onkeydown = function(evt) {
  evt = evt || window.event;
  if (evt.keyCode == 9) {
    setTimeout(function() { console.log(document.activeElement) }, 100);
  }
};
```

Then leave the console open and tab through the page, it will print out the active elements.

You'll see that the CU logo is actually getting selected as expected.

**Comment by aaronsamuel137** [comment]
Good enough for now. If you need enhancements to this please make a separate, specific issue. Closing.
